deps/libuv

```diff
@@ -0,0 +1 @@
+/Users/tmpvar/work/c/libuv
```

Comment:
@tmpvar, is this needed for the OSX build?  If so, can it use an abreviation like `~/` instead of `/Users/tmpvar`?